README.md

```diff
@@ -0,0 +1,16 @@
+# hocs-cdlz-document-extractor
+
+[![CodeQL](https://github.com/UKHomeOffice/hocs-cdlz-document-extractor/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/UKHomeOffice/hocs-cdlz-document-extractor/actions/workflows/codeql-analysis.yml)
+
+
+## Versioning
+
+For versioning this project uses SemVer.
+
+## Authors
+
+This project is authored by the Home Office.
+
+## License
+
+This project is licensed under the MIT license. For details please see [License](LICENSE)
```

Comment:
We should bring across the License file?

Reply:
It was already added when the repo was created: https://github.com/UKHomeOffice/hocs-cdlz-document-extractor/blob/main/LICENSE